crypto/dkg_test.go

```diff
@@ -429,14 +438,14 @@ func (proc *testDKGProcessor) invalidShareSend(dest int, data []byte) {
 		panic("invalid share send not supported")
 	}
 
-	newMsg := &message{proc.current, proc.msgType, data}
+	newMsg := &message{proc.current, proc.protocol, private, data}
 	// check destination
 	if (dest < recipients) || (proc.current < recipients && dest < recipients+1) ||
 		(proc.malicious == invalidComplaintAnswerBroadcast && dest == proc.dkg.Size()-1) {
 		// choose a random reason for an invalid share
 		coin := mrand.Intn(100)
```

Comment:
```suggestion
		coin := mrand.Intn(5)
```
To avoid doing `coin%5` later

Reply:
It's a code that evolved and didn't get cleaned. Thanks for flagging 👌